zuul.d/jobs.yml

```diff
@@ -0,0 +1,9 @@
+---
+- job:
```

Comment:
You need to specific the playbook to run:
  run: playbooks/ansible-test-sanity/run.yaml

https://docs.openstack.org/infra/zuul/user/config.html#attr-job.run

Reply:
Added `run: playbooks/ansible-test-sanity/run.yml`